tests/net/buf_simple/testcase.yaml

```diff
@@ -0,0 +1,5 @@
+common:
+  tags: test_framework net buf
```

Comment:
That's a good question actually. Something tells me that `test_framework` is tag that supposed to be used for testing ZTEST test framework (`tests/ztest/`), but it started to be used in other places as well. But as you said, I added this tag, because it is in other bluetooth unit tests. I'll ask on discord